drivers/clock/sunxi-ccu.c

```diff
@@ -10,48 +10,69 @@
 #include <drivers/clock.h>
 #include <drivers/clock/sunxi-ccu.h>
 
+static struct sunxi_ccu_clock *
```

Comment:
Please add a doxygen block to the header for this function.

Reply:
Done